Connection.cs

```diff
@@ -0,0 +1,42 @@
+﻿using InventoryManagementSystem.ProductInfo;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace InventorySystem
+{
+    public static class Connection
```

Comment:
Since this is responsible for production operations, I prefer to rename this to `ProductRepository`

Reply:
This doesn't have to be static, this would mean one client will be initialized for the whole project, this would causes with async calls with some  projects such as web apps. So lets not make it as a singleton, and move `Static` from the props below. Every consumer will just create a new object of this with new client for their own needs.